verticapy/connect.py

```diff
@@ -512,6 +519,9 @@ def verticalab_connection():
         "password": "",
         "database": "demo",
         "backup_server_node": ["localhost"],
-        "session_label": "verticapy",
+        "session_label": "verticapy-"
+        + verticapy.__version__
+        + "-"
+        + str(verticapy.options["identifier"]),
```

Comment:
Ditto!

Reply:
Btw verticapython client is doing the same.

Reply:
Do you mean in their `_generate_session_label()` function?
https://github.com/vertica/vertica-python/blob/16ce5b020b8b65460be48f8f7e12d0fb55d79669/vertica_python/vertica/connection.py#L246

Perhaps, it is not clear to me what you want to achieve from this "session_lable". Btw, I think it would be nice if you make a similar function for generating session labels instead of duplicating your logic.

Reply:
Ah, you may not need a function because you have the logic only in verticapy.options.